frontend/apps/reader/modules/readermenu.lua

```diff
@@ -165,20 +150,44 @@ function ReaderMenu:setUpdateItemTable()
                 },
                 {
                     text = _("Auto stretch this book's cover image in screensaver"),
```

Comment:
I would remove "Auto" from all this, and name this item "Stretch book cover to fit screen".

Also (this is just a thought while reading - if there are too much risky changes, feel free to ignore it, as the whole thing still works fine):
There and everywhere, I would replace all occurences of `proportional` with `stretch_cover` for clarity (including your helper fonction name).
(Not really sure there if `proportional_screensaver` is set to true or false when `Auto stretch `is enabled. But as there was previously this confusion on words - stretch, proportional - autostrecth actually doing autoscale, and scaling (to preserve aspect ratio) being proportional, it's a bit confusing and not clear how things do... Woud feel clearer to just use "stretch" in code, function, names, settings, when we are not keeping aspect ratio - and nothing when we do the default of keeping aspect ratio.)